packages/tmate-ubus/files/usr/libexec/rpcd/tmate

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env lua
+--[[
+  Copyright (C) 2013-2020 LibreMesh.org
+  This is free software, licensed under the GNU AFFERO GENERAL PUBLIC LICENSE Version 3
+
+]]--
+
+local ubus = require "ubus"
+local json = require 'luci.jsonc'
+local utils = require 'lime.utils'
+
+local TMATE_SOCK = "/tmp/tmate.sock"
+
+local conn = ubus.connect()
+if not conn then
+    error("Failed to connect to ubus")
+end
+
+function unix_socket_listening(name)
+   return "" ~= utils.unsafe_shell("netstat -xl | grep "..TMATE_SOCK.." 2>/dev/null")
+end
+
+local function get_session(msg)
+    if unix_socket_listening(TMATE_SOCK) then
+        local rw_ssh = utils.unsafe_shell("tmate -S "..TMATE_SOCK.." display -p ''#{tmate_ssh}|head -c -1")
+        local ro_ssh = utils.unsafe_shell("tmate -S "..TMATE_SOCK.." display -p ''#{tmate_ssh_ro}|head -c -1")
+        local clients = utils.unsafe_shell("tmate -S "..TMATE_SOCK.." display -p ''#{tmate_num_clients}|head -c -1")
+
+        utils.printJson({status = "ok", session = { rw_ssh = rw_ssh, ro_ssh = ro_ssh, clients = clients}})
+    else
+        utils.printJson({status = "error", msg = "no session"})
```

Comment:
I duplicate my previous comment about the API here to keep the feedback in a single place:

Reserve status error for unhappy paths. We could use 
{status="ok", session="no initialized"}
As session=nil will end up in a json without that field: {"status": "ok"}